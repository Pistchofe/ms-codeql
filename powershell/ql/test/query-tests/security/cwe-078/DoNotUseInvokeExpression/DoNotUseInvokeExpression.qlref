queries/security/cwe-078/DoNotUseInvokeExpression.ql