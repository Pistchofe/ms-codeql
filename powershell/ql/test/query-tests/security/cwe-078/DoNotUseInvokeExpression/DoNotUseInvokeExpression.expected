| test.ps1:2:1:2:26 | Call to Invoke-Expression | Do not use Invoke-Expression. It is a command injection risk. |
